### ISSUE TITLE
Tab rename colors

### ISSUE THREAD

**Issue by Korb** [winghongchan/dark-docs#31]
![2024-12-11_20-35-08](https://github.com/user-attachments/assets/0bbdc6db-b13b-4073-a382-a335d72fe436)

> Foreground: `#444746`
> Background: `#1F1F1F`
> Contrast ratio: 1.755:1
> 1.4.3 Contrast (Minimum) (AA)
> ❌ Fail for large and regular text
> 1.4.6 Contrast (Enhanced) (AAA)
> ❌ Fail for large and regular text
[WCAG 2.1](https://www.tpgi.com/web-content-accessibility-guidelines-wcag-2-1/)

Dark Docs 2024.11.28
Stylus 1.5.51 (August 22, 2024)
Mozilla Firefox 134.0b8 (64-bit)

Upd. +
![2024-12-11_20-48-48](https://github.com/user-attachments/assets/71b21399-bf5e-4882-932b-978767b3610e)

**Comment by winghongchan** [comment]
Dark Docs v2024.12.21 has been released with a fix for this issue and others. Please try it out, and close this issue if it looks okay now.
